files/README.md

```diff
@@ -12,7 +12,7 @@ This checksum is used to create a file tree where each package will reside, with
 
 ex:
 
- sha256sum **476e0**cdac6bc757dd2b78bacc1325323b09c45ecb41d4562deec2a1c7c148405 my-package_1.2.3_all.deb
+ sha256sum **476e**0cdac6bc757dd2b78bacc1325323b09c45ecb41d4562deec2a1c7c148405 my-package_1.2.3_all.deb
```

Comment:
What is this change?

Reply:
The file tree is composed of the first _four_ characters of the checksum. The markdown was highlighting the first five.